cinema/urls.py

```diff
@@ -1,4 +1,6 @@
+from django.conf.urls.static import static
 from django.urls import path, include
+
```

Comment:
There is no need for a blank line here - `django` and `rest_framework` belong to the same import group